src/main/java/piano/keyboard/keyboardaudio/Key.java

```diff
@@ -4,14 +4,22 @@
 
 public class Key
 {
+	String[] noteNames = new String[] { "C", "C#", "D", "D#", "E", "F", "F#", "G", "G#", "A", "A#", "B" };
```

Comment:
This should be static final. Otherwise it gets copied in every Key

Reply:
fixed in most recent commit